packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -1,128 +1,75 @@
-/**
- * https://www.builder.io/c/docs/custom-components-children
- * src/components/CustomTabs.tsx
- */
-
 import {
   Blocks,
   BuilderBlock,
   RegisteredComponent,
 } from '@builder.io/sdk-react';
 import { useState } from 'react';
 
-type Tab = {
-  tabName: string;
-  children: BuilderBlock[];
-};
 interface TabProps {
-  tabList: Tab[];
+  tabList: { tabName: string; blocks: BuilderBlock[] }[];
   builderBlock: BuilderBlock;
 }
 
-const CustomTabs = (props: TabProps) => {
+export const CustomTabs = ({ tabList, builderBlock }: TabProps) => {
   const [activeTab, setActiveTab] = useState(0);
 
   return (
-    <div>
-      <h2>Custom Component with editable regions</h2>
-
-      <div>
-        <div>
-          {/* 
-            The tabList[] prop is an array that represents the tabs.
-            Each tab has a tabName, which is displayed as the button label,
-            and children, which are the content blocks rendered inside the tab.
-            This comes from the inputs declared inside your custom component.
-            For more details: https://www.builder.io/c/docs/custom-components-input-types
-          */}
-          {props.tabList?.map((tab, index) => (
+    <>
+      {tabList?.length > 0 && (
```

Comment:
you can make this component even simpler by moving this condition outside the return:

```tsx

export const CustomTabs = ({ tabList, builderBlock }: TabProps) => {
  const [activeTab, setActiveTab] = useState(0);

  if (tablist.length === 0) { return null; }

  return ( //...
```